github/orgs_properties.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CustomProperty represents an organization custom property object.
+type CustomProperty struct {
+	PropertyName *string `json:"property_name,omitempty"`
+	// Possible values for ValueType are: string, single_select
+	ValueType     string   `json:"value_type"`
+	Required      *bool    `json:"required,omitempty"`
+	DefaultValue  *string  `json:"default_value,omitempty"`
+	Description   *string  `json:"description,omitempty"`
+	AllowedValues []string `json:"allowed_values,omitempty"`
+}
+
+// RepoCustomPropertyValue represents a repository custom property value.
+type RepoCustomPropertyValue struct {
+	RepositoryID       int64                  `json:"repository_id"`
+	RepositoryName     string                 `json:"repository_name"`
+	RepositoryFullName string                 `json:"repository_full_name"`
+	Properties         []*CustomPropertyValue `json:"properties"`
+}
+
+// CustomPropertyValue represents a custom property value.
+type CustomPropertyValue struct {
+	PropertyName string `json:"property_name"`
+	Value        string `json:"value"`
```

Comment:
Looking at the [official documentation](https://docs.github.com/en/rest/orgs/custom-properties?apiVersion=2022-11-28#list-custom-property-values-for-organization-repositories), it looks like `value` can be `null`, so let's please make this field nullable.

The "Response schema" tab says this:

```json
      "properties": {
        "type": "array",
        "items": {
          "title": "Custom Property Value",
          "description": "Custom property name and associated value",
          "type": "object",
          "properties": {
            "property_name": {
              "type": "string",
              "description": "The name of the property"
            },
            "value": {
              "type": [
                "string",
                "null"
              ],
              "description": "The value assigned to the property"
            }
          },
          "required": [
            "property_name",
            "value"
          ]
        },
```

```suggestion
	Value        *string `json:"value,omitempty"`
```